client/src/app/contact/page.tsx

```diff
@@ -0,0 +1,41 @@
+"use client"
+import Ph4 from '@/assets/ph4-contact.svg';
+import Image from 'next/image';
+
+const contact = () => {
+    return (
+        <div className="flex flex-col  w-full md:flex-row">
+            <div className="flex md:flex-row flex-col-reverse md:w-screen">
+                <div className="md:w-2/3 p-5 md:p-10 leading-7 md:leading-10">
+                    <p className="text-4xl md:text-6xl text-[#2e4e82] mb-3">Let's Connect</p>
```

Comment:
Never use this as Let's, use this like: `Let&apos;s` else build will fail.

Change everywhere you are using apos